core/src/main/java/com/linecorp/armeria/client/logging/LoggingClient.java

```diff
@@ -79,6 +81,7 @@ public static LoggingClientBuilder builder() {
         return new LoggingClientBuilder();
     }
 
+    private final Logger activeLogger;
```

Comment:
Could be just `logger`?

Reply:
fixed
I also change a default logger name from logger to defaultLogger.